ocs_ci/helpers/managed_services.py

```diff
@@ -28,21 +28,17 @@ def verify_provider_topology():
     6. Verify OSD count
     7. Verify OSD CPU and memory
 
+    Args:
+        size(str): cluster size
```

Comment:
Please add information about what happens if no size is provided.

Reply:
done